src/components/layout/data/hooks.js

```diff
@@ -14,15 +12,17 @@ export const useStylesForCustomBrandColors = (enterpriseConfig) => {
         return undefined;
       }
 
-      const { branding } = enterpriseConfig;
+      const colors = {
+        white: getComputedStyle(document.documentElement).getPropertyValue('--pgn-color-white'),
+        dark: getComputedStyle(document.documentElement).getPropertyValue('--pgn-color-dark'),
+      };
```

Comment:
[nit/suggestion] Would it make sense to create a util function of sorts to call when we want to get the `colors` object, so that we don't need to call `.getPropertyValue` for the same CSS variables in multiple filters? Perhaps a bit more DRY?

Reply:
As per our conversation, this will be a fast follow-up PR 👍🏽